Danger-Swift/Sources/DangerDependencies/Fake.swift

```diff
@@ -0,0 +1 @@
+/// Used by implementers of WeTransfer-iOS-CI in the Package.swift
```

Comment:
It looks like `Fake` is not defined in this file. Is this something to be implemented in another PR?

Reply:
You're right, this one could be removed. Did that!